client/src/keep_alive/reqrep.rs

```diff
@@ -0,0 +1,87 @@
+use super::backoff_strategy::*;
+use super::helpers::is_recoverable_error;
+use crate::request_reply::{Replier, Requestor};
+use crate::traits::KeepAliveStream;
+use futures::Future;
+use selium_std::errors::QuicError;
+use selium_std::errors::Result;
+use selium_std::traits::codec::{MessageDecoder, MessageEncoder};
+use std::fmt::Debug;
+
+pub struct KeepAliveReqRep<T> {
```

Comment:
1. Should this also be named `KeepAlive` for consistency?
2. You're missing the `doc(hidden)` annotation.

Reply:
Yep, good point! We can avoid collisions via the namespace 👌

Reply:
Added both changes 👍